### ISSUE TITLE
Team update - Jun 28, 2023

### ISSUE THREAD

**Issue by trallard** [Quansight-Labs/jupyter-a11y-mgmt#221]
This is a @Quansight-Labs/czi-a11y-grant team sync 🎉🎉🎉! This is a way for the Team Members to provide status reports on what they've been up to this week and request help and attention for things they are working on. This issue will be closed at the end of the day.

_Copy and paste the template below, and answer questions as you wish!_

<details>
<summary>Response Template</summary>

```md
**Thanks I'd like to give 🙌**
- So-and-so helped me out a lot with foo...

**Updates from last week :heavy_check_mark:**
- I worked towards deliverable: <link-to-deliverable>

**Challenges I faced and things I'd like assistance with 🙏**
- I had a hard time figuring out ...

**My availability for next week**
- I'll be off on foo day...

**Important items for discussion 💬**
- I have a question about goal <link-to-goal>
- Can @foo give a comment on issue #NN ?
- I opened #NN for discussion, please chime in
- I need to discuss foo
```

</details>

#### :mag: Needs Triage

_The [Needs Triage issues](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/labels/status%3A%20needs%20triage) require an initial assessment and labeling._

No issues need triage! 🎉

### :dart: Project boards

Please make sure the boards reflect the current status of the project.

[Jupyter a11y grant project](https://github.com/orgs/Quansight-Labs/projects/5/views/5)

**Comment by steff456** [comment]
**Updates from last week :heavy_check_mark:**
- Waiting review for notebook authoring guidelines https://github.com/jupyter/accessibility/pull/127
- Merged for the ARIA and WCAG intro https://github.com/jupyter/accessibility/pull/126
- Made the first release of JupyterLab-accessible-themes - https://pypi.org/project/jupyterlab-accessible-themes/
- Sent invites to Tania for PyPI maintainers
- Created recipe for the package - https://github.com/conda-forge/staged-recipes/pull/23183
- Opened and merged https://github.com/Quansight-Labs/jupyterlab-accessible-themes/pull/58
- Add review changes in PR for Github dark - https://github.com/Quansight-Labs/jupyterlab-accessible-themes/pull/53
- Made the internship plan - https://docs.google.com/document/d/19Cr468MmiV0Lq68A_f5rLVO4-mD590P9AmjjgioyLVQ/edit?usp=sharing
- Made PR for the 400% Zoom audit documentation - https://github.com/jupyter/accessibility/pull/133

**Challenges I faced and things I'd like assistance with 🙏**
- I'm ok

**My availability for next week**
- I'm here

**Important items for discussion 💬**
- Do we want the intern to be in our sync meetings? Or should we only invite him to the group a11y meeting?
- I still don't have any reply for external feedback about the notebook authoring guidelines, I need to do some thinking to see how we can move that PR to something that we want to have in the docs

**Comment by isabela-pf** [comment]
**Thanks I'd like to give 🙌** 

Thanks to the whole team for their patience with me as I get back to the project.

**Updates from last week :heavy_check_mark:**

- #184 I picked up writing the results again this week. Still aiming for an end of week PR.
- #147 The [second test results](https://github.com/Iota-School/notebooks-for-all/blob/main/user-tests/2-content/results.md) and [test 3 planning](https://github.com/Iota-School/notebooks-for-all/blob/main/user-tests/3-cell_structure/planning.md) (tests not run) were reviewed and merged. This means I just need to link to these results elsewhere in the ecosystem to finish the project.
- #173 picked up the [jupyter/accessibility #127 for notebook authoring practices](https://github.com/jupyter/accessibility/pull/127) to give another review based on verbal feedback. This is still in progress.

**Challenges I faced and things I'd like assistance with 🙏**
- I don't have anything right now.

**My availability for next week**
- I will be here all week.

**Important items for discussion 💬**
- I want to check in on my next steps with everyone else to make sure I didn't miss and direction changes in the meantime. Please let me know if I'm blocking you.

**Comment by gabalafou** [comment]
**Thanks I'd like to give 🙌**
- I think I might have already said thanks for this somewhere else, but thanks again anyway to @trallard and @steff456 for helping me brainstorm the cell focus styling issue for my notebook tab trap PR. I finally responded to review comments on that PR.
- Thanks to @isabela-pf for pushing back against my "command palette as skip link" idea. Critique is critical to making better decisions. Your feedback is very much welcome and appreciated.

**Updates from last week :heavy_check_mark:**
- Created issue #225
- Created issue https://github.com/jupyterlab/jupyterlab/issues/14765
- Created draft PR https://github.com/jupyterlab/jupyterlab/pull/14768
- Finished review and helped push fix to https://github.com/jupyterlab/jupyterlab/pull/14597
- Finally responded to review comments on https://github.com/jupyterlab/jupyterlab/pull/14115

**Challenges I faced and things I'd like assistance with 🙏**
- I'm not sure I can handle this many different things at once in Jupyter land.
- Reviewing the accessibility PRs has been so much more time-consuming than I expected.

**My availability for next week**
- I'll be off tomorrow, Friday, June 30.
- I will be here all next week.

**Important items for discussion 💬**
- Somebody at QuantStack will be working on keyboard navigation issues

**Comment by gabalafou** [comment]
Notes:

@isabela-pf 
- she's back!
- writing up more results from JupyterLab user testing, stretch goal: end of week pr
- will be submitting PRs to create links to the STScI work - grant deliverable has two parts: 1. make results public 2. link to results
- wondering if anything happened during Camp Quansight with creating an accessibility notebook; whether any thing happened there that needs to be taken into account for the grant documentation deliverable

**Comment by trallard** [comment]
@steff456 
	- Stephannie reached out to a couple of people individually; we need to try and do an "open" call at Quansight to reach out to folks
	- the checklist seems usable to verify/audit notebooks rather than being a resource that folks could use to write better or more accessible notebooks
	- need to debug the build and release of jupyterlab-accessible-themes
	- internship plan looks good - suggestion to try Gitpod for docs if needing to avoid setting a development environment
	- will work on fixing issues in https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/56

@gabalafou 
	- will focus on the grant deliverables 
	- was planning to work on ARIA improvements for notebooks; this will be put on hold in favour of grant deliverables
	- got back to his TAB trap PR
